app/build.gradle

```diff
@@ -11,12 +12,36 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
 
 def apiKey = getValueFromFileOrEnv(APP_TOKEN_ENV_VAR_NAME, APP_TOKEN_FILE, APP_TOKEN_KEY, ERROR_MESSAGE)
 
-def sdkVersion = '3.0.0'
-def versionCodeNumber = 11
+def sdkVersion = '4.0.0'
+def versionCodeNumber = 400
+
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.6.4'
+    }
+}
+
+dexcount {
```

Comment:
not sure if it's useful for a sample app? We probably need another test app which uses gradle external import. but for integrator is better to have a simple sample app to use as reference.

Reply:
I see. Removed this.
Regarding another app for DEX counting, let's discuss that